pages/ksphere/kommander/1.0/release-notes/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 0
+excerpt: View release-specific information for Kommander
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+## Release Notes
+
+### Version v1.0.1 May 14 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.16.0  |
+| **Maximum**        | 1.16.x  |
+| **Default**        | 1.16.8  |
+
+#### Fixed Bugs
+
+- AWS VPC route tables not removed when deploying Konvoy cluster from Kommander.
+
+#### Changes
+
+- Add 1.16.8 to selectable Kubernetes versions.
```

Comment:
I wonder if it would be useful for support/sales/customers to include component versions in release notes? We'd also have these info inside of engineering, but maybe its worth calling them out here?

```suggestion
- Add 1.16.8 to selectable Kubernetes versions.


#### Component Versions:

- Helm Chart: `0.6.12` -> `0.6.20`
- KCL: `0.5.6` -> `0.5.7`
- UI: `3.31.2` -> `3.58.0`
- kommander-thanos: `0.1.13` -> `0.1.14`
- kommander-karma: `0.3.9`
- kubeaddons-catalog: `0.1.6`
- grafana: `4.5.1`
```

Reply:
we do this for DCOS, Dispatch, Konvoy. So I say yes.